tool/site/client-react/tests/pages/__snapshots__/prescriptionSearchPage.spec.tsx.snap

```diff
@@ -3,14 +3,14 @@
 exports[`Clicking back from the detail search results returns to the summary search 1`] = `
 "<h1 class=\\"nhsuk-label-wrapper\\"><label class=\\"nhsuk-label nhsuk-label--xl\\">Search Results</label></h1>
 <div class=\\"nhsuk-table-responsive\\">
-  <table class=\\"nhsuk-table sc-jRQBWg jFVpJk\\">
+  <table class=\\"nhsuk-table sc-gKclnd cgRkdO\\">
```

Comment:
not really sure why all these things have changed? dont know if it makes a difference really?

Reply:
I wouldn't try to determine the diffs here, if you want to see what's changed you can check out the branch and use jest to see the difference between the snapshots by getting the old ones from master and running the test. Summary: "Return Prescription" prescription action now displays on this page when a prescription is with a dispenser and "Release prescription" see `prescriptionSummaryList` component under `prescription-tracker`